### DISCUSSION TITLE
VRM and remoteconsole shows the current oposite

### DISCUSSION THREAD

**Discussion by koch2907** [Louisvdw/dbus-serialbattery#348] [Q&A] [answered]

**Comment by Louisvdw** [comment, marked as answer]
Edit utils.py and change it to
> INVERT_CURRENT_MEASUREMENT = -1

**Comment by koch2907** [comment]
Thanks a Lot. now is Working perfekt. Other ideer. I use 4 batteries parallel With 280ah. But cerbo can Look only in one Batterie i Sawed. So the current is Splitting in 4parts. IT will BE interesting to add all 4 Batterie Packs to one mother Pack and this mother Pack says the Sich and the current. What you think about my ideer.BrHansSent from Outlook for AndroidVon: Louis Van Der Walt ***@***.***>Gesendet: Donnerstag, 15. Dezember 2022, 12:46An: Louisvdw/dbus-serialbattery ***@***.***>Cc: koch2907 ***@***.***>; Author ***@***.***>Betreff: Re: [Louisvdw/dbus-serialbattery] VRM and remoteconsole shows the current oposite (Discussion #348)
Edit utils.py and change it to

INVERT_CURRENT_MEASUREMENT = -1

**Reply by Louisvdw** [reply]
See #8
